bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TeaserImpl.java

```diff
@@ -114,6 +114,11 @@
      */
     private boolean titleHidden = false;
 
+    /**
+     * Flag indicating if the title type should be hidden.
+     */
+    private boolean titleTypeHidden = false;
```

Comment:
@cjelger to be backward compatible the toggle should be enabled by default, so shouldn't this be set to` true` ?

Reply:
Unfortunately this doesn't work, because when the field is not checked in the policy, the editor doesn't save the field as `false` in the JCR, but removes it completely so it ends up being `true` if we set `true` by default. Do you maybe know how to enforce that a `false` value is stored?

Reply:
Found it, there is an `uncheckedValue` property. I'll fix that.